xbmc/guilib/TextureBundleXBT.cpp

```diff
@@ -26,10 +26,8 @@
 #ifdef TARGET_WINDOWS_DESKTOP
 #ifdef NDEBUG
 #pragma comment(lib,"lzo2.lib")
-#elif defined _WIN64
-#pragma comment(lib, "lzo2d.lib")
 #else
-#pragma comment(lib, "lzo2-no_idb.lib")
+#pragma comment(lib, "lzo2d.lib")
```

Comment:
This has to be handled by the build system.

Reply:
yeah that would be better, that's for another PR though

Reply:
no, remove the hack in this PR